packages/demo-react-native/App/Config/ReactotronConfig.js

```diff
@@ -28,4 +28,6 @@ if (__DEV__) {
   Reactotron.clear()
 }
 
+Reactotron.onCustomCommand('test', () => console.log('This is an example'))
```

Comment:
did you want to `console.tron.log` this instead?  I'm ok if you don't want to.

Reply:
Yeah, I can do that. Not used to the whole `console.tron` thing because I just slap `Reactotron` on `console.log` (which is super simple now thanks to you!). Updating now.